### DISCUSSION TITLE
Can you please provide a vue3 example?

### DISCUSSION THREAD

**Discussion by rubanraj7** [diadal/universal-social-auth#22] [Q&A]
I'm having a hard time following the readme that has both vue2 and vue3 instructions.

**Comment by diadal** [comment]
[https://github.com/diadal/universal-social-auth-test-nuxt](https://github.com/diadal/universal-social-auth-test-nuxt)
https://github.com/diadal/universal-social-auth-test-vu2

**Reply by rubanraj7** [reply]
Both of these seem to be using vue2

**Reply by diadal** [reply]
you can share your code where you have an issue, is the same setup for vue3
